client/source/components/RR/tests/RRdata.js

```diff
@@ -0,0 +1,674 @@
+      helpfulness: 0,
+      photos: []
+    },
+    {
+      review_id: 841285,
+      rating: 2,
+      summary: 'Worst ever!',
+      recommend: false,
+      response: null,
+      body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+      date: '2021-09-21T00:00:00.000Z',
+      reviewer_name: 'chesterTheTester',
+      helpfulness: 0,
+      photos: []
+    },
+    {
+      review_id: 645247,
+      rating: 4,
+      summary: 'Animi odio facilis perspiciatis sunt vel deleniti explicabo.',
+      recommend: true,
+      response: null,
+      body: 'Reiciendis soluta dolore voluptates et facilis aut. Aut esse possimus minus dolores cum. Tempore autem sint culpa beatae quod explicabo nihil et tempora.',
+      date: '2020-12-05T00:00:00.000Z',
+      reviewer_name: 'Molly24',
+      helpfulness: 0,
+      photos: [
+        {
+          id: 1236149,
+          url: 'https://images.unsplash.com/photo-1553830591-2f39e38a013c?ixlib=rb-1.2.1&auto=format&fit=crop&w=2760&q=80'
+        },
+        {
+          id: 1236150,
+          url: 'https://images.unsplash.com/photo-1513531926349-466f15ec8cc7?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80'
+        },
+        {
+          id: 1236151,
+          url: 'https://images.unsplash.com/photo-1474494819794-90f9664b530d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1652&q=80'
+        }
+      ]
+    },
+    {
+      review_id: 645225,
+      rating: 2,
+      summary: 'Repellat distinctio est non id natus autem.',
+      recommend: true,
+      response: null,
+      body: 'Et nihil et nam. Accusamus sed dolore atque dignissimos nulla. Omnis occaecati facere quasi dolorum et aut.',
+      date: '2021-07-08T00:00:00.000Z',
+      reviewer_name: 'Arjun97',
+      helpfulness: 0,
+      photos: [
+        {
+          id: 1236195,
+          url: 'https://images.unsplash.com/photo-1517720359744-6d12f8a09b10?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1567&q=80'
+        },
+        {
+          id: 1236196,
+          url: 'https://images.unsplash.com/photo-1558422504-3d17542c1799?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80'
+        },
+        {
+          id: 1236197,
+          url: 'https://images.unsplash.com/photo-1534011546717-407bced4d25c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2734&q=80'
+        }
+      ]
+    }
+  ]
+};
+
+export default reviews;
```

Comment:
Do you need a 674 line long block of test data? What about just 3 or 4 reviews to switch between?

Reply:
Updated to 4 reviews